stemcell_builder/stages/bosh_rust/apply.sh

```
@@ -0,0 +1,25 @@
+base_dir=$(readlink -nf "$(dirname "${0}")/../..")
+source "${base_dir}/lib/prelude_apply.bash"
+source "${base_dir}/lib/prelude_bosh.bash"
+
+rust_version="1.96.1"
+
+# rustup verifies downloaded toolchain integrity via its own signing chain; TLS on sh.rustup.rs is the trust anchor for the installer itself
+rustup_installer=$(mktemp "${chroot}/tmp/rustup-init.XXXXXX")
+curl_five_times "${rustup_installer}" "https://sh.rustup.rs"
+
+run_in_chroot "${chroot}" "$(cat <<SCRIPT
+  installer=\$(basename ${rustup_installer#${chroot}})
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)